src/wp-includes/revision.php

```diff
@@ -516,6 +516,53 @@ function wp_get_post_revisions( $post_id = 0, $args = null ) {
 	return $revisions;
 }
 
+/**
+ * Returns last revisions of specified post and count of total revisions.
+ *
+ * @since 6.1.0
+ *
+ * @param int|WP_Post $post Optional. Post ID or post object. Default is the global $post.
```

Comment:
```suggestion
 * @param null|int|WP_Post $post Optional. Post ID or post object. null, false, 0 and other PHP falsey values return the current global post .
```

Reply:
4eb17be18cdf7b2cf8e4dd4e4511dda76a5e9a11 was made a while ago to document this type of `$post` argument consistently. 

Let's stick to the standard (although I think a few differences have slipped in).

Reply:
i updated it